### ISSUE TITLE
Ts fix

### ISSUE THREAD

**Issue by IRediTOTO** [baileyherbert/envato.js#12]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `envato@3.0.7` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/envato/dist/types/api.d.ts b/node_modules/envato/dist/types/api.d.ts
index b63775a..bc46f13 100644
--- a/node_modules/envato/dist/types/api.d.ts
+++ b/node_modules/envato/dist/types/api.d.ts
@@ -68,6 +68,12 @@ export interface Item {
         version: string;
         description: string;
     };
+    wordpress_plugin_metadata:{
+        plugin_name: string;
+        author_name: string;
+        version: string;
+        description: string;
+    },
     rating: number;
     rating_count: number;
     published_at: Date;
@@ -115,6 +121,7 @@ export interface Item {
         };
         live_site?: {
             url: string;
+            href:string
         };
         icon_preview?: {
             icon_url: string;
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by baileyherbert** [comment]
Hi @IRediTOTO! Can you share an example API request where you found `href` under the `previews.live_site` field? As far as I can see it's always called `url`.

**Comment by baileyherbert** [comment]
Nevermind, I found it. They use the undocumented `url` key when you're searching for items, and use `href` otherwise. I'll get this fixed right now, thanks!

**Comment by baileyherbert** [comment]
Fixed in `v3.0.8` - thanks! 🎉
